enso/experiment/tfidf.py

```diff
@@ -10,7 +10,6 @@
 from enso.experiment import ClassificationExperiment
 from enso.experiment.k_centers import KCentersAlgorithm
 from enso.experiment.grid_search import GridSearch
-from enso.experiment.gmean import GMeanAlgorithm
```

Comment:
So there is a class below `TfidfGMean` that requires this. Probably best to just remove it?

Reply:
Yup will do